tweak: Убрал скринтипы со стен/окон и исправления названий карт в хабе Byond

littleboobs opened the pull request:
## Что этот ПР делает
- Убрал скринтипы со всех `turf/simulated`, что надо было сделать с самого начала, поскольку смысла в скринтипах на стены нет, как нет и в скринтипах на пол. Также убрал с `/obj/structure/window`.
- Добавил `english_station_name`, который используется для хаба Бульона, чтобы не было `?????`. Также добавил кнопку в админ сикреты для переименования его, например, для кастомных/ивент карт, идею подкинул Роден. По умолчанию равно `null` и если не переопределяется, то принимает название `station_name`.
- При инициализации в логах отображается `english_station_name`.
- Вернул английские название датумов темы Лаваленда, чтобы не мозолило глаза при инициализации.

## Почему это хорошо для игры
Скринтипы не засоряют экран. Карта корректно отображается в хабе.

littleboobs committed:
NO_SCREENTIPS

littleboobs committed:
????? station_name fix

littleboobs committed:
lavaland datum names

littleboobs committed:
window NO_SCREENTIPS

ROdenFL commented on code/__HELPERS/names.dm:
```diff
@@ -49,10 +49,25 @@ GLOBAL_VAR(station_name)
 
 /proc/update_world_name()
+	/// Replacing Russian station_name with English station_name, because the Byond hub does not work with Cyrillic.
+	var/static/list/station_translations = list(
+		"ИСН Керберос" = "NSS Kerberos",
+		"ИСН Селестион" = "NSS Selestion",
+		"ИСН Фаррагус" = "NSS Farragus",
+		"ИСН Кибериада" = "NSS Cyberiad",
+		"ИСН Нова" = "NSS Nova",
+		"ШОН Мальта" = "NMC Malta",
+		"ИСН Туррим" = "NSS Turrim"
+	)
```
Наверн стоит всеже в датуме мапы сделать ченить Аля en_name или куда-нибудь в глобал вынести этот список. Иначе выходит как то через чур независимо, в плане добавления новых карт и изменения названий старых

littleboobs replied:
Да я думал об этом, но хз. Мб на че-то типо en_name реально надо поменять. :(

ROdenFL replied:
Можешь сделать его по умолчанию идентичным обычному гейм, а для карт отображающихся в зале прописать пендоский вариант

ROdenFL replied:
В хабе*

littleboobs committed:
static/list --> english_station_name

littleboobs committed:
ложь

ROdenFL commented:
Я бы сделал так, что если инглиш нейм не прописано, то равняется обычному нейм где нить в ините датума мапы, тогда везде можно было бы сразу переменную инглиш нейм юзать, без проверок на его наличие, но такой вариант тоже рабочий наверно

littleboobs committed:
Админ кнопка на ренейм в хабе и фиксы

Добавил кнопку для админов, позволяющую менять название станции в Byond hub. Теперь change_station_name() не обновляет имя в Byond hub.
